R/cassette_class.R

```diff
@@ -366,100 +366,86 @@ Cassette <- R6::R6Class(
     #' @description record an http interaction (doesn't write to disk)
     #' @param x a crul, httr, or httr2 response object, with the request at `$request`
     #' @return nothing returned
-    record_http_interaction = function(x) {
-      int <- self$make_http_interaction(x)
-      vcr_log_sprintf(
-        "Recorded HTTP interaction: %s => %s",
-        request_summary(int$request),
-        response_summary(int$response)
-      )
-      self$new_recorded_interactions <- c(
-        self$new_recorded_interactions,
-        list(int)
-      )
-    },
-
-    #' @description Are there any new recorded interactions?
-    #' @return logical
-    any_new_recorded_interactions = function() {
-      length(self$new_recorded_interactions) != 0
-    },
-
-    #' @description Make a request-response pairs
-    #' @param x A crul, httr, or httr2 response object, with the request at `$request`
-    #' @return A list with request and response.
-    make_http_interaction = function(x) {
+    record_http_interaction = function(response, request) {
```

Comment:
Wouldn't it be more consistent for the order of the param inputs to match those of the outputs? Maybe it doesn't matter too much since it's not user facing

Reply:
Oh yeah, good call.